datasets/wise-allsky.yaml

```diff
@@ -0,0 +1,28 @@
+Name: All-Sky Data | Wide-field Infrared Survey Explorer (WISE)
+Description: The All-Sky Release includes all data taken during the WISE full cryogenic mission phase, 7 January 2010 to 6 August 2010, in 4-bands (3.4, 4.6, 12, 22 microns, i.e., W1, W2, W3, W4) that were processed with improved calibrations and reduction algorithms. The Single-exposure Image Sets consist of 1,491,686 calibrated 1024x1024 pix @2.75"/pix FITS image sets for the individual 7.7 sec (W1 and W2) and 8.8 sec (W3 and W4) WISE survey exposures. Each image set consists of four intensity images, noise maps, and bit-masks indicating pixel use status, one for each WISE band.
+Documentation: https://irsa.ipac.caltech.edu/Missions/wise.html
+Contact: https://irsa.ipac.caltech.edu/docs/help_desk.html
+ManagedBy: "NASA/IPAC Infrared Science Archive ([IRSA](https://irsa.ipac.caltech.edu)) at Caltech"
+UpdateFrequency: The All-Sky Data Release has been finalized and will not be updated.
+Tags:
+  - astronomy
+  - imaging
+  - satellite imagery
+  - survey
+License: https://irsa.ipac.caltech.edu/data_use_terms.html
+Citation: "If you use the All-Sky Single-exposure Image Sets, please cite the Digital Object Identifier (DOI): https://www.ipac.caltech.edu/doi/irsa/10.26131/IRSA152. In addition, follow the [All-Sky acknowledgement guidelines](https://wise2.ipac.caltech.edu/docs/release/allsky/expsup/sec1_6b.html) and the [IRSA acknowledgement guidelines](https://irsa.ipac.caltech.edu/ack.html)."
+Resources:
+  - Description: All-Sky Single-exposure Image Sets
```

Comment:
Ya, more than one resource can be registered for the same dataset, but ours only need the one. Since the two descriptions sit next to each other on the webpage it seems unnecessary to repeat it in both places (though I didn't change AllWISE here, just fixed the formatting). But you're right, a little more info in the resource-level description might be good. I'll add what you suggested.